pkg/kvstore/etcd.go

```diff
@@ -723,6 +727,9 @@ func (e *etcdClient) LockPath(ctx context.Context, path string) (KVLocker, error
 		return nil, Hint(err)
 	}
 
+	defer func(duration *spanstat.SpanStat) {
+		increaseMetric(path, metricSet, "Lock", duration.EndError(err).Total(), err)
```

Comment:
Was thinking about appending/prepending a `Mutex` to get `LockMutex` but feel like both works equally.

Reply:
I agree. The current naming makes sense to me.